config/locales/en/course/assessment/submission/submissions.yml

```diff
@@ -76,8 +74,7 @@ en:
           submission:
             graded_not_published_warning: >
               The grading and experience points are in a draft state, and is currently not viewable
-              by the student. You can publish all draft grades by clicking the 'Publish Grades'
-              button at the top of this page.
+              by the student.
```

Comment:
"The gradings and experience points are in a draft state and cannot be seen by the student. Publish all grades by clicking the 'Publish Grades' button on the top right side of the page."